docs/movement-node/run/manul/suzuka-full-node.service.template

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Suzuka Full Node
+After=network.target
+
```

Comment:
type: manul -> manual

Reply:
> type: manul -> manual

I look so many times at that typo last evening. It looked strange and just this morning I realised :)) and then you mention it also